iati/core/resources/test_data/202/valid_unique.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
 
 <root_element>
-  <iati-identifier>This is a unique identifier</iati-identifier><!-- iati-identifier does not replicate -->
+  <iati-identifier>This is a unique identifier</iati-identifier><!-- elements do not replicate -->
```

Comment:
The mix of IATI-specific and descriptive-for-the-test element names is somewhat confusing.